src/cl_sii/dte/data_models.py

```diff
@@ -811,7 +811,12 @@ def validate_referencias_numero_linea_ref_order(cls, v: object) -> object:
         if isinstance(v, Sequence):
             for idx, referencia in enumerate(v, start=1):
                 if referencia.numero_linea_ref != idx:
-                    raise ValueError("items must be ordered according to their 'numero_linea_ref'")
+                    raise ValueError(
+                        "items must be ordered according to their 'numero_linea_ref'. "
+                        f"Expected index value: {idx}, "
+                        f"actual numero linea ref: {referencia.numero_linea_ref}. "
+                        f"referencias: {v}"
```

Comment:
We only need the complete sequence of `numero_linea_refs`.

```suggestion
                        f"All numero_linea_refs: {', '.join(ref.numero_linea_ref for ref in v)}"
```